_posts/2021-08-18-researching-the-capabilities_of_av_preservation_in_archivematica.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Researching the Capabilities of Audiovisual Preservation in Archivematica
+date: 2021-08-18T11:30:00
+author: Christian Balistreri
+layout: post
+categories:
+- Collaboration
+- Digitization
+- Software and Systems
+- Digital Preservation
+tags:
+- L. Jeffrey Selznick School of Film Preservation RAC Fellowship
+- Audiovisual
+- Film
+- Video
+- Audio
+- audiovisual digitization
+- preservation
+- Artefactual
+- Archivematica
+excerpt_separator <!--more-->
```

Comment:
It doesn't look like the front matter is displaying correctly. Is it the period in "L. Jeffrey..."?

Reply:
```suggestion
excerpt_separator: <!--more-->
```